### PR TITLE
Added support for changed cookie names

### LINKED ISSUE #5: Not working when you change SESSION_COOKIE_NAME
In Django we can set the session cookie name by setting `SESSION_COOKIE_NAME` in Django settings. Also you can change CSRF cookie name with `CSRF_COOKIE_NAME`.

But django-cookies-samesite will not work because `'sessionid'` and `'csrftoken'` is hard-coded in middleware (unless we add them inside `SESSION_COOKIE_SAMESITE_KEYS`).

### PR DESCRIPTION
**Pull request by antoinelb**
fixes #5

### CODE REVIEW

**Comment by antoinelb** [comment]
Closes #5

**Comment by antoinelb** [comment]
While waiting for comments for this PR, I put a fork on pypi at https://pypi.org/project/django-cookies-samesite-fork/. I'll take it down once this PR is merged.

**Comment by jotes** [comment]
@antoinelb 
hey, thanks for your contribution!
I have only one question: I see some changes that are unrelated to the issue.
Did you use `black` on all of those files?

File: setup.py [hunk 1]
```diff
@@ -14,60 +14,62 @@ def get_version(*file_paths):
...
-    description="""Middleware which sets SameSite flag for session and csrf cookies in legacy versions of Django.""",
-    long_description=readme + '\n\n' + history,
-    author='Jarek "jotes" Śmiejczak',
-    author_email='poke@jotes.work',
-    url='https://github.com/jotes/django-cookies-samesite',
-    packages=[
-        'django_cookies_samesite',
-    ],
+    description="Middleware which sets SameSite flag for session and csrf "
+    "cookies in legacy versions of Django. Forked from "
```
Comment (jotes) [inline setup.py]: I guess you keep some changes from your fork in your PR. Can you revert them? I'll merge the PR immediately after it. I want to add a file dedicated to all contributors.
Thanks in advance!

Comment (antoinelb) [reply setup.py]: All done!

Comment (jotes) [reply setup.py]: @antoinelb YOU are breathtaking! Thanks :dancing_men:

**Pull request merged**